decisions/release-schedule.md

```diff
@@ -0,0 +1,38 @@
+# Release Schedule
+
+## Decision
+Starting Q4 2024, a major version of the @skyscanner/backpack-web package is released approximately every 3 months, typically at the end of each quarter.
+
+### Current Release Schedule
+
+### 2024-2025
```

Comment:
do we want to keep this up to date?

Reply:
but this is open source and GC is internal, which may or may not be relevant here, well I guess even if it gets out of date it doesn't matter much as is for illustrative purposes

Reply:
It's hard, requires constant maintenance - I wonder if we should build something into the release process to update these tables?